src/site.config.ts

```diff
@@ -22,6 +22,7 @@ export const siteConfig = {
     backendApiVersion: process.env.NEXT_PUBLIC_BACKEND_API_VERSION,
     frontendServicePort: process.env.FRONTEND_SERVICE_PORT,
     frontendServiceInterface: process.env.FRONTEND_SERVICE_INTERFACE,
+    tiptapAppId: process.env.NEXT_PUBLIC_TIPTAP_APP_ID,
```

Comment:
@calebbourg Should this be build time scope or public runtime scope? Just want to double check since it's a sensitive variable.

https://nextjs.org/docs/app/building-your-application/deploying#environment-variables

Reply:
@jhodapp what's our pattern for having this injected at build time?

Reply:
The other variables that don't have `NEXT_PUBLIC_*` in front of them follow this pattern. You can see how they get injected in our [docker-compose.yaml file here](https://github.com/refactor-group/refactor-platform-rs/blob/8852758ce3dd5c95950b5aaf6b236072a1b24940/docker-compose.yaml#L64) from the backend. But if you set them directly here in the .env, they'll be part of the environment, just not exposed to Next.js.

Reply:
updated in d04813764cb45024fbab8a8a13343cbdd7c1c7bf

Reply:
```suggestion
    tiptapAppId: process.env.TIPTAP_APP_ID,
```